components/transfer/TransferOld.vue

```diff
@@ -0,0 +1,432 @@
+<template>
+  <section>
+    <Loader v-model="isLoading" :status="status" />
+    <nuxt-link
+      v-if="$route.query.target"
+      :to="`/${urlPrefix}/u/${correctAddress}`"
+      class="pl-4 is-flex is-align-items-center">
+      <NeoIcon icon="chevron-left" class="mr-2" />
+      {{ $t('teleport.artistProfile') }}
+    </nuxt-link>
+    <p class="title is-size-3">
+      {{ $t('transfer') }} {{ unit }}
+      <span class="has-text-primary">${{ getCurrentTokenValue(unit) }}</span>
+    </p>
+
+    <NeoField>
+      <Auth />
+    </NeoField>
+
+    <div v-if="targets && hasBlockExplorer" class="mb-3">
+      {{ $t('teleport.donationSentTo') }}
+      <a
+        v-for="target in targets"
+        :key="target"
+        :href="addressExplorerUrl(target)"
+        target="_blank"
+        rel="nofollow noopener noreferrer"
+        class="has-text-weight-bold">
+        <Identity ref="identity" :address="target" show-onchain-identity />
+      </a>
+    </div>
+
+    <div class="is-flex is-align-items-center">
+      <NeoField>
+        {{ $t('general.balance') }}
+        <Money :value="balance" inline />
+      </NeoField>
+    </div>
+
+    <div
+      v-for="(destinationAddress, index) in destinationAddresses"
+      :key="destinationAddress">
+      <div class="is-flex">
+        <NeoField class="is-flex-grow-1">
+          <AddressInput v-model="destinationAddresses[index]" :strict="false" />
+        </NeoField>
+        <b-button
+          v-show="index == destinationAddresses.length - 1"
+          type="is-primary"
+          size="is-small"
+          icon-left="plus"
+          class="ml-2 mt-2"
+          outlined
+          @click="addAddress">
+          Add
+        </b-button>
```

Comment:
I think @roiLeo  is working on removing b-button
you shouldn't use it

Reply:
#6449

Reply:
Hey, this component `TransferOld` is totally a copy of the original transfer component. I have done nothing changes to this file. After we finish the redesign of the transfer page, we could delete this component.

By the way, I have replaced `b-button` with `NeoButton` on the redesigned transfer component.

Reply:
Oh yeah, sorry. I thought I was looking at the new component